app/experimenter/experiments/models.py

```diff
@@ -377,17 +381,38 @@ def has_normandy_info(self):
         return self.normandy_slug or self.normandy_id
 
     @property
-    def normandy_api_recipe_url(self):
-        if self.normandy_id:
-            return settings.NORMANDY_API_RECIPE_URL.format(id=self.normandy_id)
+    def format_dc_normandy_urls(self):
+        # returns a list of dictionaries containing D.C. and Normandy
+        # urls for the main normandy id and other normandy ids if they exist
+        normandy_recipe_url = settings.NORMANDY_API_RECIPE_URL
+        delivery_console_url = settings.DELIVERY_CONSOLE_RECIPE_URL
 
```

Comment:
Had to make these variables because the Black didn't like a lot of the long lines in this method here...